resources/content/config-content/haproxy/content/etc/haproxy/haproxy.cfg.ftl

```diff
@@ -12,6 +12,9 @@ global
 	user haproxy
 	group haproxy
 	daemon
+	<#if haproxyConfig?? && haproxyConfig.global??>
```

Comment:
for future reference, freemarker has a shorter syntax for this I believe it is `(haproxyConfig.global)??`